mypy/test/helpers.py

```diff
@@ -327,3 +330,30 @@ def parse_options(program_text: str, testcase: DataDrivenTestCase,
         options.python_version = testcase_pyversion(testcase.file, testcase.name)
 
     return options
+
+
+def split_lines(*streams: bytes) -> List[str]:
+    """Returns a single list of string lines from the byte streams in args."""
+    return [
+        s.rstrip('\n\r')
```

Comment:
minor: `splitlines()` is already going to remove the line endings (only keeps them with `splitlines(True)`)

```pycon
>>> 'foo\r\nbaz\nwomp\r'.splitlines()
['foo', 'baz', 'womp']
```

Reply:
(ah, this is just moved code -- feel free to disregard! not a regression!)

Reply:
AIUI, we don't want trailing newlines to generate a an empty string entry, so I believe this is actually intentional.

```
>>>'a\nb\nc\n\n'.splitlines()
['a', 'b', 'c', '']
```

Reply:
It's too late for that then, the strip is applying *after* `splitlines`:

```pycon
>>> def split_lines(*streams: bytes) -> List[str]:
...     """Returns a single list of string lines from the byte streams in args."""
...     return [
...         s.rstrip('\n\r')
...         for stream in streams
...         for s in stream.decode('utf8').splitlines()
...     ]
... 
>>> def split_lines2(*streams: bytes) -> List[str]:
...      return [
...          s
...          for stream in streams
...          for s in stream.decode('utf8').splitlines()
...      ]
... 
>>> split_lines(b'foo\nbar\nbaz\n\n')
['foo', 'bar', 'baz', '']
>>> split_lines2(b'foo\nbar\nbaz\n\n')
['foo', 'bar', 'baz', '']
```

Reply:
Ah, good point. I will go ahead and remove it while this PR touches it.